pkgs/development/python-modules/spectral-cube/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, astropy
+, radio_beam }:
+
+buildPythonPackage rec {
+  pname = "spectral-cube";
+  version = "0.4.3";
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.